lib/templates/keyinfo.tpl.xml.js

```diff
@@ -1,3 +1,4 @@
+module.exports = `
 <KeyInfo xmlns="http://www.w3.org/2000/09/xmldsig#">
   <e:EncryptedKey xmlns:e="http://www.w3.org/2001/04/xmlenc#">
     <e:EncryptionMethod Algorithm="<%= keyEncryptionMethod %>">
```

Comment:
While you're at it, do you think you can change this to use template literals since we'll be moving off node 4? 
eg. https://github.com/auth0/node-xml-encryption/pull/40/files#diff-50e3aa130a4f97a42ee2cf111c7b1d9dR5

Let me know your thoughts, thanks!

Reply:
I've gone through and:
- removed old travis configuration for node 4 and 6
- replaced ejs usage with template literals. I've used the escape-html library to provide html escaping where it was relied upon from ejs.